UCSD/campus-mobile
# Add Native Direction Routes to Map

## Summary

This PR implements native route drawing in the maps view for the app when the user presses the
walking directions button on the app, which is added to the app in PR #1230 .


<img width="330" alt="image" src="https://user-images.githubusercontent.com/28168427/109032855-e8473300-767a-11eb-8ac7-66d2c520d2b2.png">

## Changelog

[General] [Add] - Native routes drawing for directions in maps view


## Test Plan


1. Press the directions button in the maps view when the search bar is empty.
    - This should take users to search view
2. Input a location into the search bar and wait for the location marker to appear in screen.
3. Press the directions button again. This will draw a route from the user's current location
to the location marker.
